dll/win32/shell32/utils.cpp

```diff
@@ -99,3 +99,134 @@ SHFindComputer(LPCITEMIDLIST pidlRoot, LPCITEMIDLIST pidlSavedSearch)
 
     return SUCCEEDED(hr);
 }
+
+static VOID Int64ToStr(LONGLONG llValue, LPWSTR pszValue)
+{
+    WCHAR szBuff[40];
+    INT ich = 0;
+#if (WINVER >= 0x0600)
```

Comment:
```suggestion
#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Reply:
Can we unhack it in the CMakeList.txt? Like we/you recently did in some other module?